src/components/HomePage/HomePage.jsx

```diff
@@ -12,7 +15,6 @@ class HomePage extends Component {
 	render() {
 		const {	increment, decrement, resetNumberToZero } = this.props;
 		return (
-			<React.Fragment>
```

Comment:
Only use a fragment when you're returning multiple elements, otherwise it's just add meta. example:
```javascript
// multiple component children with no parent, you'll use Fragment here to avoid adding an unneeded div to the DOM
return (
  <React.Fragment>
     <div />
     <div />
     <div />
  </React.Fragment>
```
alternate example:
```javascript
// Don't need Fragment because you already have one parent of the component's children that you're/we're using here.
return (
  <div>
     <div />
     <div />
     <div />
  </div>
);
```